### PR TITLE
create goodpractice::goodpractice alias from goodpractice::gp, resolves #52

### LINKED ISSUE #52: rename main function to goodpractice
Hi @gaborcsardi, 
I can implement this if you'd like but I think the main function in the package should be named `goodpractice`, same as the package. 
What do you think? 
Thanks,
Marcel

### PR DESCRIPTION
**Pull request by LiNk-NY**
Changes include: 

* alias function creation and exporting
* authorship information in R format
* reformatted package documentation

Feel free to change it at your leisure. 

Regards,
Marcel

### CODE REVIEW

**Comment by LiNk-NY** [comment]
@gaborcsardi I ran the check on your package and it wasn't passing. I wasn't sure why.

**Comment by LiNk-NY** [comment]
@gaborcsardi Any thoughts on this? Thanks.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,8 @@
 Package: goodpractice
 Title: Advice on R Package Building
-Version: 1.0.0
-Author: Gabor Csardi
+Version: 1.0.1
```
Comment (gaborcsardi) [inline DESCRIPTION]: Don't change the version number, please.

File: DESCRIPTION [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+Authors@R: person("Gabor", "Csardi", email = "csardi.gabor@gmail.com",
+    role = c("aut", "cre"))
```
Comment (gaborcsardi) [inline DESCRIPTION]: If you add `Authors@R`, then the `Maintainer` field is not needed.

File: R/package.R [hunk 3]
```diff
@@ -1,10 +1,8 @@
...
 #' Give advice about good practices when building R packages. Advice includes
 #' functions and syntax to avoid, package structure, code complexity, code
 #' formatting, etc.
-#'
-#' @docType package
-#' @name goodpractice
-NULL
+#' @name goodpractice-package
+#' @aliases NULL
+"_PACKAGE"
```
Comment (gaborcsardi) [inline R/package.R]: why _PACKAGE? Can you at least remove the alias to NULL and add back the `@docType`?

File: DESCRIPTION (hunk 1 above)
Comment (LiNk-NY) [inline DESCRIPTION]: Version bumps shouldn't be a problem but if you want to keep it the same, I'll leave it as is.

File: R/package.R (hunk 3 above)
Comment (LiNk-NY) [inline R/package.R]: The `NULL` alias avoids the conflicts between the `goodpractice` function page and the package name page. The "_PACKAGE" takes care of adding `@docType` parameter. This is the more updated version of documenting packages, AFAIK.

File: DESCRIPTION (hunk 2 above)
Comment (LiNk-NY) [inline DESCRIPTION]: I totally missed this line. Thanks for pointing it out.

**Comment by LiNk-NY** [comment]
Any updates on implementing this?

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**